### DISCUSSION TITLE
[woocommerce_my_account] no muestra desplegables

### DISCUSSION THREAD

**Discussion by Omariuschile** [whooohq/whq-woocommerce-chilexpress-shipping#218] [General]
Estimados
El plugin funciona perfectamente en la página [woocommerce_checkout]
Pero al ingresar a [woocommerce_my_account] los campos de Región y Comuna no se muestran con lista desplegable, solo como campo de texto.
Para habilitar es tema de configuración o se debe modificar el código?
Muchas gracias

**Comment by EstebanForge** [comment]
Estimado,

Tienes toda la razón. Es en estas rutas:
http://wpchilexpress.whooohq.com/my-account/edit-address/facturacion/
http://wpchilexpress.whooohq.com/my-account/edit-address/envio/
Que es donde se edita las direcciones de facturación y envío.

Pero...

WC de todos modos debe pasar por el carro de compra y/o el checkout al hacer una compra, que es el momento en donde se calcula el costo de envío y se cobra el total.
Aunque coloques una ciudad que no existe (en Chile) en esas dos rutas, al momento de pasar por el carro o checkout, el plugin de todos modos limpiará ese valor incorrecto y requerirá que pongas una ciudad (o región) válida, de la lista que Chilexpress entrega vía API.

Habiendo dicho eso,

Si en esas dos rutas en my-account NO hay un cálculo de precio y costo de envío (que es lo que este plugin quiere resolver vía Chilexpress), la verdad, no se si debamos meternos ahí.

¿O no?
